static/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -763,6 +802,7 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
         <DividerContainer>
           {this.renderDivider(dividerHandlerChildrenProps)}
           {this.renderErrorBadge(errors)}
+          {this.renderEmbeddedTransactionsBadge(transactions)}
```

Comment:
I assume the case where there's is both an error and a child transaction will be left as a TODO item?

Reply:
Yep. I'm going to follow up in another pull request that will include some hover state effects to be able to support this.